tools/releng/launch-build

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
I'm assuming you don't want to leave this in?  The `TRACE=1` envvar already exists below to turn this on.

Reply:
Thanks - I forgot that.